### PR TITLE
Ac updates

### PR DESCRIPTION
**Pull request by adamclark64**

### CODE REVIEW

File: app/stylesheets/defaults/typography.scss [hunk 1]
```diff
@@ -9,7 +9,8 @@ body {
   color: $black;
   font-family: Helvetica, Arial, sans-serif;
   font-size: $font-size-base; // 16px
-  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: subpixel-antialiased;
+  -webkit-font-smoothing: subpixel-antialiased;
```
Comment (gtwalford) [inline app/stylesheets/defaults/typography.scss]: let's leave this as antialiased -but do add the moz browser extension

File: webpack/dev.config.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ var config = {
   devtool: 'eval-source-map',
   entry: {
     app: [
+      'babel-polyfill', // from YOU IE crashing fix
```
Comment (gtwalford) [inline webpack/dev.config.js]: remove YOU comment

Comment (gtwalford) [inline webpack/dev.config.js]: You also need to update the app file too where babel-polyfill is imported

**Review by gtwalford — CHANGES_REQUESTED** [review]

File: app/stylesheets/defaults/typography.scss (hunk 1 above)
Comment (adamclark64) [inline app/stylesheets/defaults/typography.scss]: sounds good d6013dd

File: webpack/dev.config.js (hunk 2 above)
Comment (adamclark64) [inline webpack/dev.config.js]: cool took out YOU specific comment 9e7b2b7 and got id of the import in app/app.js 3846a67

**Review by gtwalford — APPROVED** [review]

**Pull request merged**